test/unit/conditions/base/rpc.test.ts

```diff
@@ -9,22 +9,4 @@ describe('validation', () => {
       _class: 'RpcCondition',
     });
   });
-
```

Comment:
In the `nucypher` code you've mentioned there are two allowed RPC methods, `balanceOf` and `eth_getBalance`. But the validation method will always throw on `balanceOf`. So the only supported method is `eth_getBalance`, and no arbitrary methods are permitted (since they would have to go into `ALLOWED_METHODS`. Am I reading this correctly?

Reply:
`balanceof` and `eth_getBalance` are in the ALLOWED_METHODS list and are the only ones that are allowed.

@KPrasch , @cygnusv I though we wanted to expand this list - see https://github.com/nucypher/tdec/issues/64? I guess not for v7.0.0?

If not for v7.0.0, @piotr-roslaniec my apologies for wasting your time.

Reply:
Hmm in that case maybe we could revert the [changes from this commit](https://github.com/nucypher/nucypher-ts/pull/220/commits/a080664b779e90cc88dc5a4900c5f41164a89511) and revisit https://github.com/nucypher/tdec/issues/64 in another PR

Reply:
Yeah that's probably good.